src/include/ipxe/errfile.h

```diff
@@ -213,6 +213,8 @@ FILE_LICENCE ( GPL2_OR_LATER_OR_UBDL );
 #define ERRFILE_usbblk		     ( ERRFILE_DRIVER | 0x00ce0000 )
 #define ERRFILE_iphone		     ( ERRFILE_DRIVER | 0x00cf0000 )
 #define ERRFILE_slirp		     ( ERRFILE_DRIVER | 0x00d00000 )
+#define ERRFILE_aqc1xx		     ( ERRFILE_DRIVER | 0x00d00000 )
```

Comment:
Already used by slirp?

Reply:
Oops. Sorry, did not notice this conflict after rebase. Thanks!

Reply:
Fixed 👌